tests/test_geometry_loader.py

```diff
@@ -49,13 +49,11 @@ def test_load(self):
         eq_(True, is_new)
         
         # We can measure the distance in kilometers between New York
-        # and Texas. This verifies that the GeoJSON shapes are
-        # imported as real-world geographies, not abstract
-        # geometries.
-        distance_func = func.ST_Distance(new_york.geo, texas_zip.geo)
+        # and Texas.
+        distance_func = func.ST_Distance_Sphere(new_york.geometry, texas_zip.geometry)
         distance_qu = self._db.query().add_columns(distance_func)
         [[distance]] = distance_qu.all()
-        eq_(2511, int(distance/1000))
+        eq_(2510, int(distance/1000))
```

Comment:
These numbers are slightly different because ST_Distance on a geography takes the shape of the Earth into account, whereas ST_Distance_Sphere assumes the Earth is a sphere. The difference doesn't matter for our purposes.